MonoGame.Framework/Microsoft/Devices/Sensors/Accelerometer.cs

```diff
@@ -0,0 +1,76 @@
+// MonoGame - Copyright (C) MonoGame Foundation, Inc
+// This file is subject to the terms and conditions defined in
+// file 'LICENSE.txt', which is part of this source code package.
+
+using System;
+using System.Runtime.InteropServices;
+using Microsoft.Xna.Framework;
+
+namespace Microsoft.Devices.Sensors
+{
+    /// <summary>
+    /// Provides Desktop applications access to the device's accelerometer sensor.
+    /// Currently stubs, for cross-platform development.
+    /// </summary>
+    public sealed partial class Accelerometer : SensorBase<AccelerometerReading>
+    {
+        /// <summary>
+        /// Gets or sets whether the device on which the application is running supports the accelerometer sensor.
+        /// </summary>
+        public static bool IsSupported => PlatformIsSupported();
+
+        /// <summary>
+        /// Gets the current state of the accelerometer. The value is a member of the SensorState enumeration.
+        /// </summary>
+        public SensorState State => PlatformSensorState();
+
+        /// <summary>
+        /// Creates a new instance of the Accelerometer object.
+        /// </summary>
+        public Accelerometer()
+        {
+            PlatformAccelerometer();
+        }
+
+        /// <summary>
+        /// Initializes the platform resources required for the accelerometer sensor.
+        /// </summary>
+        static void Initialize()
+        {
+            PlatformInitialize();
+        }
+
+        void ActivityPaused(object sender, EventArgs eventArgs)
```

Comment:
I don't think these should be in the this file, but should be in the Android one as they are Android specific

Reply:
@dellis1972 I've left them in Android and removed them everywhere else. Could you please double check the Compass implementation on Android, just to be sure that's correct?